common/functions/src/scalars/arithmetics/arithmetic_modulo.rs

```diff
@@ -45,44 +45,35 @@ pub struct ArithmeticModule<T, D, M, R> {
 impl<T, D, M, R> ArithmeticTrait for ArithmeticModule<T, D, M, R>
 where
     T: DFPrimitiveType + AsPrimitive<M>,
-    D: DFPrimitiveType + AsPrimitive<M>,
+    D: DFPrimitiveType + AsPrimitive<M> + num::One,
     R: DFPrimitiveType,
     M: DFPrimitiveType + num::Zero + AsPrimitive<R> + Rem<Output = M>,
     u8: AsPrimitive<R>,
     u16: AsPrimitive<R>,
     u32: AsPrimitive<R>,
     u64: AsPrimitive<R>,
     DFPrimitiveArray<R>: IntoSeries,
+    R: Into<DataValue>,
 {
     fn arithmetic(columns: &DataColumnsWithField) -> Result<DataColumn> {
-        let lhs = columns[0].column().to_minimal_array()?;
-        let rhs = columns[1].column().to_minimal_array()?;
-        let lhs: &DFPrimitiveArray<T> = lhs.static_cast();
-        let rhs: &DFPrimitiveArray<D> = rhs.static_cast();
-        let result_type = R::data_type();
-
-        let need_check = result_type.is_integer();
-
-        let result: DataColumn = try_binary_arithmetic_helper! {
-            lhs,
-            rhs,
+        let need_check = R::data_type().is_integer();
```

Comment:
For `div`, `modulo`.

`D::default()` may cause div by zero error.

Reply:
> For `div`, `modulo`.
> 
> `D::default()` may cause div by zero error.

Yes, I use d::one()